### PR TITLE
Update tree.rst

### PR DESCRIPTION
**Pull request by stosh15x**
Added examples to the generateTreeList options: keyPath and valuePath

### CODE REVIEW

File: en/core-libraries/behaviors/tree.rst [hunk 1]
```diff
@@ -422,8 +422,8 @@ are a few more tree-orientated permutations at your disposal.
     .. php:method:: generateTreeList ($conditions=null, $keyPath=null, $valuePath=null, $spacer= '_', $recursive=null)
 
     :param $conditions: Uses the same conditional options as find().
-    :param $keyPath: Path to the field to use for the key.
-    :param $valuePath: Path to the field to use for the label.
+    :param $keyPath: Path to the field to use for the key, i.e. “{n}.Post.id”.
```
Comment (dereuromark) [inline en/core-libraries/behaviors/tree.rst]: Is `“` a valid char?

Comment (stosh15x) [inline en/core-libraries/behaviors/tree.rst]: The examples were copied from lines 469 and 470

Comment (markstory) [inline en/core-libraries/behaviors/tree.rst]: smartquotes make sphinx sad. Can these be changed to `"` (ascii double quote).

Comment (stosh15x) [inline en/core-libraries/behaviors/tree.rst]: updated and re commited

**Pull request merged**